From 31ed081268c4e3e17cf4c46919e74980c65cab00 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Mon, 10 Aug 2015 10:17:56 +0800
Subject: [PATCH] MA-6965 Remove CONFIG_IMX6_USB_CHARGER for android kernel

Align the change in below commit from imx_3.14.y branch:
  commit e497a6a24d5a7d5d7fa481eaaf99369028e3590f
  Author: Li Jun <jun.li@freescale.com>
  Date:   Tue Aug 4 18:57:20 2015 +0800

    MLK-11319-4 ARM: imx_v7_defconfig: remove CONFIG_IMX6_USB_CHARGER

    Remove CONFIG_IMX6_USB_CHARGER since it's no longer used.

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arch/arm/configs/imx_v7_android_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/imx_v7_android_defconfig b/arch/arm/configs/imx_v7_android_defconfig
index 66b3e47..cf4a186 100644
--- a/arch/arm/configs/imx_v7_android_defconfig
+++ b/arch/arm/configs/imx_v7_android_defconfig
@@ -338,7 +338,6 @@ CONFIG_GPIO_SYSFS=y
 CONFIG_GPIO_MAX732X=y
 CONFIG_GPIO_74X164=y
 CONFIG_SABRESD_MAX8903=y
-CONFIG_IMX6_USB_CHARGER=y
 CONFIG_SENSORS_MAX17135=y
 CONFIG_SENSORS_MAG3110=y
 CONFIG_THERMAL=y
-- 
1.8.0

